File: Examples/Misc/VB/Examples/MiscExamples/FormMain.vb
Public Class FormMain

    Public Sub New()

        InitializeComponent()

        Me.Text = "NetOffice Misc Examples in Visual Basic"
        LoadExamples()

    End Sub

    Private Sub LoadExamples()

        LoadExample(New Example01())
        LoadExample(New Example02())
        LoadExample(New Example03())

    End Sub

End Class

File: Examples/Misc/C#/Examples/MiscExamples/FormMain.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

using ExampleBase;

namespace MiscExamplesCS4
{
    public partial class FormMain : FormBase
    {
        public FormMain()
        {
            InitializeComponent();

            this.Text = "NetOffice Misc Examples in C#";
            LoadExamples();
        }

        private void LoadExamples()
        {
            LoadExample(new Example01());
            LoadExample(new Example02());
            LoadExample(new Example03());
        }
    }
}
